tools/traffic/workers/key_handler.go

```diff
@@ -0,0 +1,7 @@
+package workers
+
+// KeyHandler is an interface describing an object that can accept unconfirmed keys.
+type KeyHandler interface {
```

Comment:
This will go away in the next PR when I implement a request from @ian-shim to use a channel to pass this data. I'm intentionally not changing it in this PR to reduce the number of merge conflicts in my feature branch.